src/main/java/edu/upf/taln/dri/common/util/GateUtil.java

```diff
@@ -351,7 +351,7 @@ public static void transferAnnotations(Document doc, String annType, String newA
 	 * @param endOffset
 	 * @return
 	 */
```

Comment:
Variable rename just to help us understand the code better. Most of the changes are just this rename being propagated.